aten/src/ATen/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(USE_ROCM)
 endif()
 
 # NB: If you edit these globs, you'll have to update setup.py package_data as well
-FILE(GLOB base_h "*.h" "detail/*.h" "cpu/*.h")
+FILE(GLOB base_h "*.h" "detail/*.h" "cpu/*.h" "cpu/vec256/*.h" "quantized/*.h")
```

Comment:
Interesting. Does this fix some recompilation problems?

Reply:
It's better to put unrelated changes like this in a separate PR

Reply:
We need native/cpu/* and other headers to be available in cpp extensions because we are going to compile templatized distribution kernels